src/ubuntu/18.04/amd64/Dockerfile

```diff
@@ -17,6 +17,7 @@ RUN apt-get update \
         sudo \
         wget \
     && rm -rf /var/lib/apt/lists/* \
+    && gem install public_suffix -v 4.0.7 \
```

Comment:
For each of these changes, it would be good to have a comment explaining why it's being done with a link to this PR.

Reply:
updated.  I'll try to research this more. Something changed and it may be just some fluke in package dependencies. It used to work without it.